stdlib/LinearAlgebra/src/LinearAlgebra.jl

```diff
@@ -355,36 +355,37 @@ control over the factorization of `B`.
 """
 rdiv!(A, B)
 
-
+copy_oftype(A::AbstractArray{T}, ::Type{T}) where {T} = copy(A)
+copy_oftype(A::AbstractArray{T,N}, ::Type{S}) where {T,N,S} = convert(AbstractArray{S,N}, A)
 
 """
-    copy_oftype(A, T)
+    copymutable_oftype(A, T)
```

Comment:
Why not `copy_mutable_oftype`  or  `copy_oftype_mutable`?

This last one is the most intuitive for me.

Reply:
I don't have any preference, but `copymutable_oftype` follows the same logic as `copy_oftype`: when `eltype(A) == T`, then in the one case you use `copy`, in the other case you use `Base.copymutable`. And the `_oftype` appendix says that the respective copy should be "of (a given) type".

Reply:
Ok. It makes sense then. Maybe we can add a reference to `Base.copymutable` in the docstring to allow readers to learn two functions at the same time? To make the logic of the name clearer.

Reply:
Actually, now that I read my own response I'm not so sure anymore. 😅 As I explained in the original commit, `copy(::AbstractArray)` calls `Base.copymutable` by default. I'll link to the `Base.copymutable` docstring, but better won't try to explain any "naming logic".